crates/holochain/src/core/queue_consumer.rs

```diff
@@ -561,6 +561,11 @@ impl TriggerReceiver {
         }
         Ok(())
     }
+
+    /// Check whether the backoff loop is paused. Will always return false if there is no backoff for this receiver.
+    pub fn is_paused(&self) -> bool {
```

Comment:
Convenience for testing, it looks like this logic hasn't been tested before because I couldn't see another way to verify pause/resume calls